en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml

```diff
@@ -1490,7 +1525,7 @@ paths:
         - API Revisions
       summary: Retrieve Revision
       description: |
-        Retrieve a revision of an API
+        Retrieve a revision of an API (This resource is not supported at the moment)
```

Comment:
Suggestions : 
1. This is resource is not supported in the current release.
2. This resource is not supported by WSO2 API Manager as of yet.

Reply:
Fixed